src/Adapter/index.js

```diff
@@ -6,7 +6,19 @@ const options = {
   valueEncoding: 'json',
 };
 
-const { union } = Gun.HAM;
+const union = function union(vertex, node, opt){
+	if(!node || !node._){ return }
+	vertex = vertex || Gun.state.to(node);
+	if(!vertex || !vertex._){ return }
+	opt = Gun.num.is(opt)? {machine: opt} : {machine: Gun.state()};
+	opt.union = Gun.obj.copy(vertex); // Slow performance.
+	if(!Gun.node.is(node, function(val, key){
+		var HAM = Gun.HAM(opt.machine, Gun.state.is(node, key), Gun.state.is(vertex, key, true), val, vertex[key]);
+		if(!HAM.incoming){ return }
+		Gun.state.to(node, key, opt.union);
+	})){ return }
+	return opt.union;
+}
```

Comment:
I feel ya. Most of my builds fail if the lint doesn't pass. A drawback I've seen is it adds another barrier to entry for contributors. This might be a good project to experiment with [prettier](https://github.com/prettier/prettier) on 🤔